src/client/render/core.h

```diff
@@ -41,6 +42,8 @@ class RenderingCore
 
 	v2f virtual_size_scale;
 	v2u32 virtual_size { 0, 0 };
+	video::IImage *screenshot = nullptr;
+	TextureBuffer *m_buffer = nullptr;
```

Comment:
name `headless_buffer`?
Nothing else has m_prefix, and it makes it clear that it's used only for headless rendering.